src/main.cpp

```diff
@@ -59,10 +59,17 @@ void Q_DECL_UNUSED initRCCIconTheme()
 
 std::unique_ptr<QCoreApplication> createApplication(int& argc, char* argv[])
 {
-    for (int i = 1; i < argc; ++i) {
-        if (!qstrcmp(argv[i], "--exportTo"))
+    const std::vector<std::string> args(argv, argv + argc);
```

Comment:
please keep the normal loop, no need for the vector nor the std::string copies (string_view would be better, but the vector is simply useless in this context)